<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>967</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>401</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>URW Bookman L</family>
     <pointsize>32</pointsize>
    </font>
   </property>
   <property name="text">
    <string>New Shipment</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>URW Bookman L</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Owner:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>URW Bookman L</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Flavor:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>180</y>
     <width>61</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="dropOwner">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>110</y>
     <width>80</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>441</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>URW Bookman L</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Number of Bags/Containers in Shipment:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="editFlavor">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>160</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="editNumBag">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>200</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>340</y>
     <width>95</width>
     <height>27</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>250</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>URW Bookman L</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Location Stored:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="dropLocation">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>250</y>
     <width>80</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
